workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,19 +49,70 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
```

Comment:
after looking at the two methods it looks like the code is nearly completely the same for `LaunchClient` and `LaunchLocalDeployment`. Try to factor it out in a generic `Launch` method in which you can specify which spatial command you are running.
The only difference I see right now are the logfile name and the logs. The logs can be made a bit more generic, while the logfile name can be passed as a parameter

Reply:
Definitely, will do.